universal/pm.md

```diff
@@ -2,32 +2,40 @@
 * [AMD CPU Power Management](../universal/pm.md#amd-cpu-power-management)
 
-Ivy Bridge and Ivy Bridge-E note: Apple dropped support for XCPM back in macOS Sierra, so XCPM is only supported between 10.8.5 and 10.11.6. Newer OSes will require the [ssdtPRgen method](../universal/pm.md#sandy-and-ivy-bridge-power-management)
+::: details Ivy Bridge and Ivy Bridge-E note
 
-* To enabled XCPM in older OSes(ie. 10.11 and older), simply add `-xcpm` to your boot-args
+Apple dropped support for XCPM back in macOS Sierra, so XCPM is only supported between 10.8.5 and 10.11.6. Newer OSes will require the [ssdtPRgen method](../universal/pm.md#sandy-and-ivy-bridge-power-management)
 
-To start, grab [IORegistryExplorer](https://github.com/khronokernel/IORegistryClone/blob/master/ioreg-302.zip) and look for `AppleACPICPU`(note if you use search, it won't show the children so clear your search once you've found the entry):
+:::
+
+::: tip
+
+To enable XCPM in 10.11 and older, simply add `-xcpm` to your boot-args.
+
+:::
+
+To start, grab [IORegistryExplorer](https://github.com/khronokernel/IORegistryClone/blob/master/ioreg-302.zip) and look for `AppleACPICPU` (note that if you use search, it won't show the children so clear the search box once you've found the entry):
 
 XCPM Present           |  Missing XCPM
 :-------------------------:|:-------------------------:
 ![](../images/post-install/pm-md/pm-working.png)  |  ![](../images/post-install/pm-md/pm-not-working.png)
 
-As you can see from the left image, we have the X86PlatformPlugin attached meaning Apple's CPU Power Management Drivers are doing their thing(Note the CPU's name does not matter, CPU names come in many variations such as CP00, CPU0, PR00, etc. What matters is that AppleACPICPU attaches to it). If you get something like to the right image, then there's likely an issue. Make sure to check the following:
+As you can see from the image on the left, we have X86PlatformPlugin attached meaning Apple's CPU power management drivers are doing their thing (note that the name of the CPU does not matter). If you see something similar to the image on the right, then there's likely an issue. Make sure to check the following:
 
 * SSDT-PLUG.**aml** is both present and enabled in your config.plist and EFI/OC/ACPI
   * If you're missing this, head to [Getting Started With ACPI](https://dortania.github.io/Getting-Started-With-ACPI) on how to make this
-* SSDT-PLUG is set to the first thread of your CPU, you can check by selecting the first CPU listed(`CP00` for our example) and make sure you have this in the properties:
+* SSDT-PLUG is set to the first thread of your CPU, you can check by selecting the first CPU listed (`CP00` for our example) and make sure you see this in the properties:
 
 ```
 plugin-type | Number | 0x1
 ```
 
-**X99 Note**:
+::: details X99 Note
 
 XCPM does not natively support Haswell-E and Broadwell-E, this means we need to spoof the CPU ID into a model that does supports XCPM:
```

Comment:
Uh. I think Haswell-E is supported by `_cpuid_set_info`, so it will not KP. Broadwell-E is not supported at all.

Better to mention this, though I do not really mind, as we do need to do the CPUID spoofing for both generations.